src/notebook.c

```diff
@@ -473,6 +496,12 @@ static void show_tab_bar_popup_menu(GdkEventButton *event, GeanyDocument *doc)
 	g_signal_connect(menu_item, "activate", G_CALLBACK(on_close_other_documents1_activate), doc);
 	gtk_widget_set_sensitive(GTK_WIDGET(menu_item), (doc != NULL));
 
+	menu_item = ui_image_menu_item_new(GTK_STOCK_CLOSE, _("Close Documents to the _Right"));
```

Comment:
Valid remark from @elextr on IRC: as tabs can be put on the right, so stack vertically, it might be useful to say "below".

Another partly valid point that was raised is that it might be possible to change RTL/LTR without affecting the effective language of the UI.  I don't think it's too important, but could be checked manually with `gtk_widget_get_direction()` and then providing the different possible strings (left/right/below -- let's not get crazy and imagine UI can be bottom-up…).